packages/botonic-plugin-flow-builder/src/action.tsx

```diff
@@ -18,31 +19,33 @@ export class FlowBuilderAction extends React.Component<FlowBuilderActionProps> {
     const flowBuilderPlugin = getFlowBuilderPlugin(request.plugins)
     const locale = flowBuilderPlugin.getLocale(request.session)
     const payload = request.input.payload
-    let targetContentId: string | undefined = payload
+    let targetNode: HtNodeWithContent | string | undefined = payload
```

Comment:
Better not to store the payload in this variable and initially make it undefined.
`let targetNode: HtNodeWithContent | string | undefined = undefined`

Reply:
As we discussed, if we enter to this action through a payload, the payload will be the targetNode's ID, so I would leave it as it is, this way we will have it already set it from the beginning of the action.